src/main/java/ml/codeboy/bukkitbootstrap/gui/MultiPageGui.java

```diff
@@ -100,6 +100,10 @@ protected String getPageName(int id) {
         return "page " + id;
     }
 
+    public void addItem(ItemStack item, AdvancedAction action) {
+        addItem(item, (Action) action);
+    }
+
```

Comment:
Same as with GUI I don't think we need that java class system does that automatically